operator/integration-tests/model-in-image/mii-sample-wrapper/README

```diff
@@ -129,3 +129,11 @@ INTROSPECTOR_DEADLINE_SECONDS ::: Introspector timeout for JRF runs
 DB_NAMESPACE ::: Namespace for Oracle DB
 
 ?DB_NODEPORT ::: Nodeport for Oracle DB
+
```

Comment:
Add IMAGE_TYPE explanation...

Reply:
And change MODEL_IMAGE_TAG explanation above to reflect its new dependency on IMAGE_TYPE